crates/marketplace/src/testing/integration.rs

```diff
@@ -187,39 +172,6 @@ mod tests {
         .await
     }
 
-    #[tokio::test(flavor = "multi_thread")]
-    #[tracing::instrument]
-    #[ignore = "slow"]
-    async fn stress_test() {
```

Comment:
We've had CI for slow tests disabled altogether because this test would often fail due to CI boxes performance. This RP re-enables other slow tests and removes this one specifically.